crates/re_types/definitions/arrow/attributes.fbs

```diff
@@ -0,0 +1,15 @@
+namespace arrow;
+
+/// Marks a union as sparse, affecting its Arrow datatype.
+///
+/// This does _not_ affect the object generated in Python or Rust, this is a pure Arrow concern!
```

Comment:
C++ is "just around the corner" so better to refer to "the objects in generated code" not to any specific language :)
Also a little bit misleading because it does affect codegen, just not the interface :thinking: